.NET/Microsoft.Recognizers.Text.Number/Extractors/BaseNumberRangeExtractor.cs

```diff
@@ -232,14 +233,28 @@ private List<ExtractResult> ExtractNumberAndOrdinalFromStr(string numberStr, boo
                 ret = ret.OrderByDescending(num => num.Length).ThenByDescending(num => num.Start).ToList();
             }
 
-            if (ret != null && isAmbiguousRangeOrFraction)
+            var removeFractionWithInConnector = false;
+
+            if (Options == NumberOptions.ExperimentalMode)
```

Comment:
This is not the correct way to check for enum flags.

Reply:
Can you also move the logic that sets removeFractionWithInConnector to its own method? This is likely to change in the future and would make the code easier to read.